pxl_scripts/update_readme.py

```diff
@@ -1,12 +1,16 @@
+# Run using python3.
+
 import os
 import yaml
 
+if os.getcwd().split("/").pop() == "pixie":
+	os.chdir("./pxl_scripts")
 
```

Comment:
For this to be a little cleaner you can probably have it take in a path to look for the scripts as an argument to the python script.

Reply:
Rather than take in an argument, I look for the root directory and append the path to the scripts.

Reply:
I know, it’s just a bit weird to have this hard coded in the script when it’s not necessary.